### PR TITLE
Update the version of code generator and regenerate

### PR DESCRIPTION
**Pull request by arjunrn**
Update dependencies:
- `client-go`
- `api`
- `apimachinery`
- `code-generator`

Regenerate code for client and informers

### CODE REVIEW

**Comment by mikkeloscar** [comment]
:+1:

**Comment by arjunrn** [comment]
:+1:

File: hack/update-codegen.sh [hunk 1]
```diff
@@ -41,7 +41,7 @@ done
 # use vendor/ as a temporary stash for code-generator.
 rm -rf "${SCRIPT_ROOT}/vendor/k8s.io/code-generator"
 rm -rf "${SCRIPT_ROOT}/vendor/k8s.io/gengo"
-git clone --branch=release-1.12 https://github.com/kubernetes/code-generator.git "${SCRIPT_ROOT}/vendor/k8s.io/code-generator"
+git clone --branch=kubernetes-1.16.3 https://github.com/kubernetes/code-generator.git "${SCRIPT_ROOT}/vendor/k8s.io/code-generator"
```
Comment (aermakov-zalando) [inline hack/update-codegen.sh]: Do we really still need this, now that k8s supports go modules?

Comment (mikkeloscar) [reply hack/update-codegen.sh]: We can do this trick: https://github.com/zalando-incubator/cluster-lifecycle-manager/blob/master/tools.go but doesn't have to be in this PR IMO.

**Comment by aermakov-zalando** [comment]
👍

File: hack/update-codegen.sh [hunk 2]
```diff
@@ -18,43 +18,25 @@ set -o errexit
...
 # generate the code with:
 # --output-base    because this script should also be able to run inside the vendor dir of
 #                  k8s.io/kubernetes. The output-base is needed for the generators to output into the vendor dir
 #                  instead of the $GOPATH directly. For normal projects this can be dropped.
-
-OUTPUT_BASE="$(dirname ${BASH_SOURCE})/"
-
-cd "${SCRIPT_ROOT}"
-"${CODEGEN_PKG}/generate-groups.sh" all \
+echo $PWD
```
Comment (mikkeloscar) [inline hack/update-codegen.sh]: nit: remove debug code.

**Comment by arjunrn** [comment]
:+1:

**Comment by aermakov-zalando** [comment]
👍

**Pull request merged**